CHANGELOG.md

```diff
@@ -10,7 +10,15 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ### Added
 
 - Adds MongoDB instrumentation support from .NET Core 3.1+.
-
+- Raname generic environment variables to include trace.
```

Comment:
I think that we need `Changed` header

Reply:
Fixed e6068984182957fc5e6fdd49f88e2735ba249c28